services/discourse/discourse.service.js

```diff
@@ -40,6 +40,8 @@ class DiscourseBase extends BaseJsonService {
 function DiscourseMetricIntegrationFactory({ metricName, property }) {
   return class DiscourseMetric extends DiscourseBase {
     static get name() {
+      // The space is needed se we get 'DiscourseTopics' rather than
```

Comment:
Thanks for adding the comment. Minor typo (needed _so_ we get..)

Reply:
Was trying to fix this via the browser but GH is crashing on me

Reply:
Updated!

Reply:
you beat me!